indico/modules/events/contributions/lists.py

```diff
@@ -80,28 +82,28 @@ def _filter_list_entries(self, query, filters):
             if status_criteria:
                 criteria.append(db.or_(*status_criteria))
 
-        if 'speakers' in filters['items']:
-            registration_join_criteria = db.and_(
+        if filtered_people := filters['items'].get('people') or filters['items'].get('speakers'):
```

Comment:
I really don't like the idea of using the walrus operator in this way (using the assigned value somewhere much later, outside the body of that particular ``if` statement

Reply:
That was quick :D You have a point, I was trying to get rid of some lines, let me know if the new changes are simpler.